JDBC/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.metacube.jdbc.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+
+	private static final String DB_URL = "jdbc:mysql://localhost:3306/library";
+	private static final String USER = "root";
+	private static final String PASSWORD = "mysql";		
+	private Connection con = null;
+	public Connection getConnection() {
+		/* register driver */
+		try {
+			Class.forName("com.mysql.jdbc.Driver");
```

Comment:
You can have a constant for JDBC driver as well as we may want to change in future then we can directly modify the constant.